lms/djangoapps/bulk_email/models.py

```diff
@@ -215,6 +216,10 @@ def render_htmltext(self, htmltext, context):
         Convert HTML text body (`htmltext`) into HTML email message using the
         stored HTML template and the provided `context` dict.
         """
+        # HTML-escape string values in the context (used for keyword substitution).
+        for key, value in context.iteritems():
```

Comment:
@cahrens: Seems ok.  Could you use `markupsafe.escape()` instead?  I think that should be our go to when we need escaping.